### ISSUE TITLE
Failed: unresolved dependency: com.typesafe.play#build-link (snapshot)

### ISSUE THREAD

**Issue by seglo** [playframework/playframework#10108]
https://travis-ci.com/playframework/playframework/jobs/293818725?utm_medium=notification&utm_source=email

```
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [warn] 	::          UNRESOLVED DEPENDENCIES         ::
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [warn] 	:: com.typesafe.play#build-link;2.8.1+23-b77889c4: not found
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [warn] 
[info] [warn] 	Note: Unresolved dependencies path:
[info] [warn] 		com.typesafe.play:build-link:2.8.1+23-b77889c4
[info] [warn] 		  +- com.typesafe.play:run-support_2.10:2.8.1+23-b77889c4
[info] [warn] 		  +- com.typesafe.play:sbt-plugin:2.8.1+23-b77889c4 (scalaVersion=2.10, sbtVersion=0.13) (/tmp/sbt_b6ecb2f3/play-position-mapper/project/plugins.sbt#L6-7)
[info] [warn] 		  +- com.typesafe.play:sbt-scripted-tools:2.8.1+23-b77889c4 (scalaVersion=2.10, sbtVersion=0.13) (/tmp/sbt_b6ecb2f3/play-position-mapper/project/plugins.sbt#L7-8)
[info] [warn] 		  +- default:play-position-mapper-build:0.1-SNAPSHOT (scalaVersion=2.10, sbtVersion=0.13)
[info] sbt.ResolveException: unresolved dependency: com.typesafe.play#build-link;2.8.1+23-b77889c4: not found
```

**Comment by raboof** [comment]
This was fixed by https://github.com/playframework/playframework/pull/10083 on master, but this failure was on 2.8.x, so I just requested a backport to 2.8.x

**Comment by seglo** [comment]
Seems to still be a problem on master.

https://travis-ci.com/github/playframework/playframework/jobs/315634652#L1500

**Comment by mkurz** [comment]
I am pretty this will be fixed by #10855

**Comment by mkurz** [comment]
Sorry, closed to early...
